packages/headless/rollup.config.js

```diff
@@ -1,4 +1,5 @@
 import resolve from '@rollup/plugin-node-resolve';
+import json from '@rollup/plugin-json';
```

Comment:
What is that plugin purpose in this scenario ?

Which JSON file do we need to bundle ?

Reply:
The JSON file is coming from the `pino` dependency, which is importing it's `package.json` to expose it's version via an exported variable.